egs/wsj/s5/steps/data/reverberate_data_dir.py

```diff
@@ -479,13 +489,14 @@ def ParseNoiseList(noise_list_file):
                               'to their original duration and added at a random point of the speech.', choices = ["background", "foreground"])
     noise_parser.add_argument('--room-linkage', type=str, default=None, help='required if isotropic, should not be specified if point-source.')
     noise_parser.add_argument('--probability', type=float, default=None, help='probability of the noise.')
-    noise_parser.add_argument('noise_file_location', type=str, help='noise file location')
+    noise_parser.add_argument('noise_file_location', type=str, help="noise file location, it can be either a filename or a piped command. "
```

Comment:
please update the comments and argument names as you are supporting rspecifiers rather than just file names.